app/views/invite_requests/status.html.erb

```diff
@@ -1,12 +1,12 @@
 <!--Descriptive page name, messages and instructions-->
 <h2 class="heading">
-  <%= ts("Invitation Request Status") %>
+  <%= t(".heading") %>
 </h2>
 
 <p>
-  <%= ts("There are currently %{count} people on the waiting list.", count: InviteRequest.count) %>
+  <%= t(".waiting_list", count: InviteRequest.count) %>
   <% if AdminSetting.current.invite_from_queue_enabled? %>
-    <%= ts("We are sending out %{invites} invitations per day.", invites: AdminSetting.current.invite_from_queue_number) %>
+    <%= t(".send_rate", invites: AdminSetting.current.invite_from_queue_number) %>
```

Comment:
Completely optional, nitpick: It would be nice if this also used count as the interpolation variable and had the one/other distinction in the locale.